main.go

```diff
@@ -12,14 +12,24 @@ func main() {
 		logger.Fatalw("missing dependency", "bin", "codefresh")
 	}
 
+	// verify if the github CLI is installed
```

Comment:
Since Codefresh and Slack are given title case
```suggestion
	// verify if the Github CLI is installed
```